### PR TITLE
[MM-55119] Replace usage of LocalizedIcon in 'fa_previous_icon.tsx' with i/span tags

### PR DESCRIPTION
**Pull request by bishalpal**
#### Summary

This PR replaces the usage of LocalizedIcon with i tag in `webapp/channels/src/components/widgets/icons/fa_previous_icon.tsx` file.

#### Ticket Link

Jira https://mattermost.atlassian.net/browse/MM-55119

Fixes : #25091 

#### Release Note

```release-note
NONE
```

### CODE REVIEW

File: webapp/channels/src/components/widgets/icons/fa_previous_icon.tsx [hunk 1]
```diff
@@ -3,27 +3,20 @@
...
     additionalClassName?: string;
 }
 
-const iconTitle = defineMessage({
-    id: 'generic_icons.previous',
-    defaultMessage: 'Previous Icon',
-});
-
 const PreviousIcon = ({additionalClassName}: Props) => {
+    const {formatMessage} = useIntl();
```
Comment (M-ZubairAhmed) [inline webapp/channels/src/components/widgets/icons/fa_previous_icon.tsx:13]: nit : empty space in between

**Review by M-ZubairAhmed — COMMENTED** [review]
You are on 🔥 with the quality PRs you have been sending out

File: webapp/channels/src/components/widgets/icons/fa_previous_icon.tsx (hunk 1 above)
Comment (bishalpal) [reply webapp/channels/src/components/widgets/icons/fa_previous_icon.tsx:13]: @M-ZubairAhmed ,
sure, I will add it.

**Review by jespino — APPROVED** [review]
Looks good to me. Thanks @bishalpal! :tada:

File: webapp/channels/src/components/widgets/icons/fa_previous_icon.tsx [hunk 2]
```diff
@@ -3,27 +3,21 @@
...
+
```
Comment (jespino) [inline webapp/channels/src/components/widgets/icons/fa_previous_icon.tsx:14]: It is not shown here but you are adding 4 spaces and the linter is not liking it. Just keep the empty line completely empty.

Comment (bishalpal) [reply webapp/channels/src/components/widgets/icons/fa_previous_icon.tsx:14]: @jespino, Sure, I will remove the spaces on line 14.

**Review by M-ZubairAhmed — APPROVED** [review]

**Review by jespino — APPROVED** [review]
Nice! thanks @bishalpal! :tada:

**Review by yasserfaraazkhan — APPROVED** [review]
tested this on server. working as expected

**Pull request merged**